docker-compose.yml

```diff
@@ -27,6 +27,35 @@ services:
       - ./pgadmin-data:/var/lib/pgadmin
     depends_on:
       - db
+  nestjs:
+    build:
+      context: .
+      dockerfile: ./Dockerfile
+    image: tusharchy/nest-and-postgres-application:latest
+    environment:
+      - DB_TYPE=${DB_TYPE}
+      - DB_HOST=${DB_HOST}
+      - DB_PORT=${DB_PORT}
+      - DB_USERNAME=${DB_USERNAME}
+      - DB_PASSWORD=${DB_PASSWORD}
+      - DB_NAME=${DB_NAME}
+      - POSTGRES_SYNC=false
```

Comment:
What is this variable for? didn't you mean to configure the sync flag by any chance ? Otherwise I would say to take it out

Reply:
I will check it out